program/src/instruction.rs

```diff
@@ -124,11 +128,14 @@ mod tests {
     #[test]
     fn serialize_initialize() {
         let cluster_type = ClusterType::Development;
+        let size = 1_000u64;
```

Comment:
Should we have a constant for the program tests? 1_000 or 1_000u64 is used in a few places.